requirements/base.in

```diff
@@ -4,7 +4,7 @@ cached_property
 click
 click-default-group
 colorlog==4.7.2
-numpy==1.20.3
+numpy
```

Comment:
I remember issues with this. Do you mind finding out the version of numpy that was installed now on your env and pin it here? Thanks!

Reply:
The issues were specifically with Pypy actually. If we use the normal Python interpreter, even the latest versions of numpy should work like a charm, so I think that this is ok!